### ISSUE TITLE
Write

### ISSUE THREAD

**Issue by ghost** [daohoangson/flutter_widget_from_html#1]
I have an idea.
What if you could have flutter html forms that send data to a server and it renders the html and then the html flutter view updates automatically.

It's basically a full round trip.

Also you could use http path links to navigate around the flutter app using a the fluoro flutter router. The schema might need to be : "app:blah/foo/bar.html for example to distinguish links from those that you want to go to a web server etc.

**Comment by daohoangson** [comment]
For that use case, webview probably works better I think? Of course we need to wait for inline webview to be implemented in Flutter...

**Comment by gedw99** [comment]
I would not use a Webview. The whole point of converting HTML to Flutter Widgets is to avoid the issues with the web. 
Anyway, you should read what i wrote in the issue again. I think its a fantastic way to do it.

**Comment by daohoangson** [comment]
Yes, it's a great idea. Something like http://jasonette.com without the need for JSON even. I will see if I can write up an example for this. Sounds pretty complicated though, hmm.

**Comment by OHeroJ** [comment]
> Yes, it's a great idea. Something like http://jasonette.com without the need for JSON even. I will see if I can write up an example for this. Sounds pretty complicated though, hmm.

like this: https://github.com/dengyin2000/dynamic_widget
